app/views/edit/build-config.html

```diff
@@ -141,254 +165,278 @@
+                                </a>
+                              </span></label>
+                              <key-value-editor
+                                entries="imageSourcePaths"
+                                key-placeholder="Source Path"
+                                key-validator="\/.*?$"
+                                value-placeholder="Destination Dir"
+                                key-validator-error-tooltip="A valid Source Path is an absolute path beginning with '/'"
+                                add-row-link="Add image source path"></key-value-editor>
                             </div>
                           </div>
 
-                          <div>
-                            <div class="form-group">
-                              <label for="buildFrom">Push To</label>
-                              <ui-select required ng-model="imageOptions.to.type" search-enabled="false">
-                                <ui-select-match>{{$select.selected | startCase}}</ui-select-match>
-                                <ui-select-choices repeat="type in pushToTypes">
-                                  {{type | startCase}}
-                                </ui-select-choices>
-                              </ui-select>
-                            </div>
-                            <div class="form-group" ng-if="imageOptions.to.type==='ImageStreamTag'">
-                              <istag-select model="imageOptions.to.imageStreamTag"
-                                            allow-custom-tag="true"></istag-select>
-                            </div>
 
-                            <div ng-if="imageOptions.to.type==='DockerImage'" class="form-group">
-                              <label for="pushToLink">Docker Image Repository</label>
-                              <div>
-                                <input class="form-control"
-                                  id="pushToLink"
-                                  name="pushToLink"
-                                  type="text"
-                                  ng-model="imageOptions.to.dockerImage"
-                                  placeholder="example: centos/ruby-20-centos7:latest"
-                                  autocorrect="off"
-                                  autocapitalize="off"
-                                  spellcheck="false"
-                                  required>
-                              </div>
+                          <div class="multiple-image-source" ng-if="sourceImages.length !== 1">
+                            <label for="imageSourceFrom">Image Source From<span class="help action-inline">
+                              <a href="">
+                                <i class="pficon pficon-info" style="cursor: help;" aria-hidden="true" data-toggle="tooltip" data-original-title="This Build Config contains more then one Image Source. To edit them use the YAML editor.">
```

Comment:
We should consistently add the cursor style as well. Some of the tooltips don't have it.